README.md

```diff
@@ -267,6 +267,23 @@ put_vars(
 
 [Java Code](https://github.com/search?type=code&q=repo:metafacture/metafacture-fix+path:FixMethod.java+"+put_vars+{")
 
+##### `to_var`
+
+Defines a single global variable that can be referenced with `$[<variableName>]` and assinges the value of the `<sourceField>`.
+
+```perl
+to_var("<sourceField>", "<variableValue>")
+```
+
+Options:
+
+- `"default"`: provide default value if source field does not exist.
```

Comment:
Drop quotes, capitalize first word after colon.

Option description should say what it _is_, not what it _does_.

```suggestion
- `default`: Default value if source field does not exist.
```

Reply:
Isn't it important to highlight that this option has to be in quotes?

Reply:
See [discussion in #354](https://github.com/metafacture/metafacture-fix/pull/356#discussion_r1799506405).

Reply:
```suggestion
- `default`: Default value if source field does not exist. The option needs to be written in quotation marks because it is a reserved word in Java. (Default: Empty string)
```